data/places/tei/4130.xml

```diff
@@ -143,6 +147,30 @@
             <ptr target="http://syriaca.org/cbss/GC2NFIZI"/>
             <citedRange unit="p">88-104</citedRange>
           </bibl>
+          <bibl xml:id="bib4130-4">
+            <title xml:lang="en" level="a">Gospel of Matthew</title>
```

Comment:
No need to add the `title`, `editor`, etc. elements here. They are permitted, but will be overwritten by the information in the CBSS bibliographic record on the website. More just for time saving you can leave these off